merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/instantiators/ActivityMapperInstantiator.java

```diff
@@ -144,4 +145,13 @@ default MethodSpec makeGetArgumentsMethod(final ActivityTypeRecord activityType)
             .reduce(CodeBlock.builder(), (x, y) -> x.add(y.build()))
             .build());
   }
+
+  static ActivityMapperInstantiator make(final ActivityDefaultsStyle style) {
```

Comment:
Did this method sneak in from a different commit? I don't see it used anywhere in this commit.

(It looks like it was meant to be in the next commit, specifically.)

Reply:
Ah, no, ignore me -- I got confused because bits were moved around in addition to being added, but this *is* used in this commit.

Reply:
All good :) I should probably make this "review as a whole" instead of by commit, some of the refactoring was layered to make things a bit easier (for example, renaming before refactoring).